tools/scripts/fuzz.sh

```diff
@@ -40,4 +40,4 @@ nice "./bazel-bin/test/fuzz/$what" \
   -dict=test/fuzz/ruby.dict \
   -artifact_prefix=fuzz_crashers/original/ \
   fuzz_corpus \
-  $FUZZ_ARGS
+  "$FUZZ_ARGS"
```

Comment:
This is a bit hacky, maybe there's a better way to do this.

Reply:
to be clear - the hacky thing is not this, but the fact that i set FUZZ_ARGS to `--` by default so that putting quotes here doesn't mess things up. Because if the default value of FUZZ_ARGS was "", then putting quotes here would mean we pass "" to the fuzz binary which isn't good (it causes it to fail with `No such file or directory ""`). `--` is parsed as "this is the end of the options" and so doesn't do anything.